cspc-operator.Dockerfile

```diff
@@ -0,0 +1,51 @@
+#
+# This Dockerfile builds a recent cspc-operator using the latest binary from
+# cspc-operator  releases.
+#
+FROM golang:1.13.6 as build
+
+ARG TARGETPLATFORM
+
+ENV GO111MODULE=on \
+  CGO_ENABLED=1 \
+  DEBIAN_FRONTEND=noninteractive \
+  PATH="/root/go/bin:${PATH}"
+
+WORKDIR /go/src/github.com/openebs/cstor-operator/
+
+RUN apt-get update && apt-get install -y make git zip
+
+COPY . .
+
+RUN export GOOS=$(echo ${TARGETPLATFORM} | cut -d / -f1) && \
+  export GOARCH=$(echo ${TARGETPLATFORM} | cut -d / -f2) && \
+  GOARM=$(echo ${TARGETPLATFORM} | cut -d / -f3 | cut -c2-) && \
+  make buildx.cspc-operator
+
+FROM alpine:3.11.5
```

Comment:
If we've compiled or binary and that gets copied into our final stage can we perhaps use a distroless or scratch image?